### ISSUE TITLE
Incorect definiton generated for localized aspects

### ISSUE THREAD

**Issue by theres** [mrbandler/cds2types#39]
```cds
aspect Foo {
    foo: localized String
}

entity Bar : Foo {
    key bar: String
}
```

will produce

```ts
export namespace Bar {
    export interface ITexts {
        locale: string;
        foo: string;
        bar: string;
    }

    export enum Entity {
        Texts = "Bar.texts"
    }

    export enum SanitizedEntity {
        Texts = "Texts"
    }
}

export interface IBar {
    foo: string;
    bar: string;
    texts: ITexts[];
    localized?: ITexts;
}

export enum Entity {
    Bar = "Bar"
}

export enum SanitizedEntity {
    Bar = "Bar"
}
```

That won't compile. It also breaks usege of common SAP aspect - `CodeList`

**Comment by thisisevanfox** [comment]
Hi @theres,

I think this issue refers to #33. With the changes made in #27 your example produces:

```ts   
export interface IBar {
    foo: string;
    bar: string;
    texts?: IBarTexts[];
    localized?: IBarTexts;
 }

export interface IBarTexts {
    locale: string;
    foo: string;
    bar: string;
}

export enum Entity {
    Bar = "Bar",
    BarTexts = "Bar.texts",
}

export enum SanitizedEntity {
    Bar = "Bar",
    BarTexts = "BarTexts",
}
```

Is this the expected outcome?

The fix will be released with the next major release.

**Comment by thisisevanfox** [comment]
Closed with v3.0.0 (#45).
